.github/workflows/publish-docs.yml

```diff
@@ -6,6 +6,8 @@ on:
   release:
     types:
       - published
+env:
+  GO_VERSION: "1.21.1"
```

Comment:
There's:

```yaml
    - name: Set up Go
      uses: actions/setup-go@v4
      with:
        go-version-file: go.mod
```

to get the version from go.mod

Reply:
Thanks! I've updated the PR